CareKitUI/CareKitUI/Shared/Extensions/Number+Extensions.swift

```diff
@@ -40,7 +44,16 @@ extension CGFloat {
 
     /// Scaled value for the current size category.
     func scaled() -> CGFloat {
-        UIFontMetrics.default.scaledValue(for: self)
+
+        #if canImport(UIKit)
+
+        return  UIFontMetrics.default.scaledValue(for: self)
+
+        #else
+
+        return 1
```

Comment:
This scale value may not be correct, made this change because I could only find `UIFontMetrics` in `UIKit`, let me know if you have an improvement on this.